collector/pkg/component/analyzer/network/protocol/testdata/dubbo/server-trace-short.yml

```diff
@@ -2,7 +2,7 @@ trace:
   key: shortData
   requests:
     -
-      name: "read"
+      name: "write"
```

Comment:
I know "write" represents "request" for clients, and "read" does it for servers. But why should it be modified here?